docs/sessions/2026-03-28-build-run-proxy-debug.md

```
@@ -0,0 +1,140 @@
+# Session: Build, Run, and Proxy Debug
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)